frontend/src/old-pages/Configure/HeadNode.tsx

```diff
@@ -147,10 +152,11 @@ function headNodeValidate() {
     clearState([...errorsPath, 'onConfigured'])
   }
 
+  valid = slurmAccountingValidateAndSetErrors()
+
   setState([...errorsPath, 'validated'], true)
 
   const config = getState(['app', 'wizard', 'config'])
-  //console.log(config);
```

Comment:
suggestion: you can also remove the `config` assignment, since it's not used anymore

Reply:
Will remove it.